### DISCUSSION TITLE
Kids Zone

### DISCUSSION THREAD

**Discussion by mayhair** [iptv-org/community#724] [Q&A]
**Channel Name:** Kids Zone
**Channel ID:** KidsZone.pk
**Country:** Pakistan
**Language:** Urdu, English
**Website:** Currently down ...
**Stream URL (optional):** xxx
**Notes (optional):** Not sure if this is satellite-only or not, and if they air Urdu and English cartoons on the same feed.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19745
